ui/apps/platform/src/Containers/Policies/PolicyPage.tsx

```diff
@@ -56,9 +56,12 @@ function PolicyPage({
 }: PolicyPageProps): ReactElement {
     const { wizardPolicy } = useSelector(wizardPolicyState);
 
+    // If wizardPolicy: ClientPolicy is correct above, then getClientWizardPolicy is unneeded below.
```

Comment:
Is it worth creating a Jira ticket to track this for later? I also noticed that there are a lot of type assertions in the `getClientWizardPolicy` function calls, so it seems like some wires are crossed here, even if we aren't getting errors now.

Edit: I see now that those assertions are modified later down in the PR

Reply:
Good idea about a task for internal tech debt.

Although more lenient checking by TypeScript and typescript-eslint might have been a mercy at the time to meet a desperate deadline, it seems like wires crossed indeed.